src/renderer/store/mutations.js

```diff
@@ -548,7 +550,34 @@ function SOCKET_RECONNECT(state) {
       market: state.currentMarket,
       isRequestSilent: true,
     });
+
+    // Ensure trade history is up-to-date on reconnect. (may have dropped some trades during disconnect)
+    this.dispatch('fetchTradeHistory', {
+      marketName: this.$store.state.currentMarket.marketName,
+      isRequestSilent: true,
+    });
+  }
+}
+
+function tradeUpdateReceived(state, tradeUpdateMsg) {
+  if (!state.tradeHistory || !state.tradeHistory.trades) {
+    return;
   }
+
+  tradeUpdateMsg.trades.forEach((trade) => {
+    const tradeMsgLength = 3;
```

Comment:
@rlucas6130 We definitely do not want to create this variable inside of an iterated function. Instead, please move this to the the top of mutations.js and name it something like TRADE_MSG_LENGTH.